src/Modules/Frontend/templates/base/Mails/Default.html.twig

```diff
@@ -1584,6 +1584,3 @@
 <div style="display:none; white-space:nowrap; font:15px courier; line-height:0;"> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </div>
 </body>
 </html>
-
```

Comment:
@daphneslootmans zin die newlines niet nodig voor een of andere domme mailclient?

Reply:
Ja, best behouden

Reply:
@bjorvack precies niet opgelost? Want ze zijn nog steeds weg de newlines.

Reply:
fixed 364d3c4681b357abe901b8cebddf20cc45967c66